projects/hide-keyboard/src/lib/hide-keyboard.directive.ts

```diff
@@ -45,7 +45,9 @@ export class HideKeyboardDirective {
 		this.el.nativeElement.readOnly = value;
 		if (this.el.nativeElement.children && this.el.nativeElement.children.length > 0) {
 			this.el.nativeElement.children[0].readOnly = value;
-			this.el.nativeElement.children[1].readOnly = value;
+      if(this.el.nativeElement.children[1] !== undefined) {
```

Comment:
We can also use this check if you would like:
`if (this.el.nativeElement.children.length > 1)`